### ISSUE TITLE
Investigate adding `base-uri`

### ISSUE THREAD

**Issue by robhudson** [mozilla/bedrock#15555]

**Comment by janbrasna** [comment]
There don't seem to be any `<base>` elements set visibly from a quick search, and I also don't recall any env settings to inject it for some deployments (e.g. don't see it being used even in `test.bedrock.nonprod.webservices.*`) so the goal is perhaps to set it to `'none'`, right?

The public facing site should be fine, question is whether Wagtail doesn't need that for anything, but the only base use I can spot is in targets for opening new windows, so a RO test-drive should surface any violations, but hopefully there would be none. 🤞

**Comment by stevejalim** [comment]
Just an FYI but `<base>` appears to be used in Wagtail's internal live preview panel (see [this image](https://docs.wagtail.org/en/v4.0.3/_images/screen13_preview_panel.png) as an example)

https://github.com/wagtail/wagtail/commit/54597bb65d70fc41f05beb6c711ffed82fcd8dc7

So if we do add it, please can we double-check that in-CMS previews still work fine.

**Comment by janbrasna** [comment]
Noticed this exact use before and it should have no impact: `<base target>` is not affected by restricting base href, and its possible vectors [were](https://t.ly/-n7LT#/w3c/webappsec-csp/issues/186) decided to be addressed [elsewhere](https://t.ly/1wYNC#/whatwg/html/issues/2655) so it seems like no target changes are planned in CSP for the foreseeable future.

**Comment by robhudson** [comment]
> There don't seem to be any <base> elements set visibly from a quick search

There aren't but this also protects if a malicious script were to dynamically add a `<base>` target to the page affecting any forms or other elements on the page that may use relative addressing.

**Comment by janbrasna** [comment]
> > There don't seem to be any  elements set visibly from a quick search
> 
> There aren't but this also protects if a malicious script were to dynamically add a `<base>`

Right, I meant it as if we need to support any pre-existing values, or a wholesale `NONE` would suffice.

I went with `'none'` in #15580 for now.

**Comment by janbrasna** [comment]
This has been in _dev_ for a week and in _prod_ for a few days. Unless there are any violations reported for this _(esp. from CMS)_, I'd include it in some tightening next week, moving a bunch of report-only rules to enforced.

**Comment by robhudson** [comment]
I'm seeing a few `base-uri` errors come through but they look to be legitimate from the QQ browser.

**Comment by janbrasna** [comment]
This has been moved to enforced policy a month ago. If there are no interesting Sentry reports I'd say this can be closed as done.
